arkit_utils/undistort_images/undistort_image_cuda.py

```diff
@@ -5,6 +5,19 @@
 import argparse
 from numba import cuda
 import tqdm
+import math
+
+global undistort_data
+undistort_data = None
+
+class UndistortData:
+    def __init__(self, crop_left, crop_top, crop_width, crop_height):
```

Comment:
duplicate code.

Reply:
merge code in [this commit](https://github.com/homee-ai/nerfstudio/pull/9/commits/027bae0be48e8acc9f4b12e6bb9e798105443020)